lib/features/composer/domain/repository/composer_repository.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:tmail_ui_user/features/composer/domain/model/email_request.dart';
+
+abstract class ComposerRepository {
+  Future<bool> sendEmail(AccountId accountId, EmailRequest emailRequest);
+
+  Future<bool> saveEmailAddress(List<EmailAddress> listEmailAddress);
+
+  Future<List<EmailAddress>> getListEmailAddress({String? word, int? limit, String? orderBy});
```

Comment:
please move it to `AutoCompleteRepository`. We can combine all datasources to `AutoCompleteRepositiry` in the future, not only in `saved` emails, but also in device contacts, and may be in remote API.
```
Future<List<EmailAddress>> getAutoComplete(AutoCompletePattern)
```